PhotoApp/MainViewController/Other/AboutViewController.swift

```diff
@@ -10,10 +10,21 @@ import UIKit
 
 class AboutViewController: UIViewController {
 
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var githubButton: UIButton!
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        self.imageView.image = UIImage.init(named: "me")
+        self.imageView.contentMode = .scaleAspectFill
         // Do any additional setup after loading the view.
     }
 
+    @IBAction func vkbuttonTaped(_ sender: Any) {
+        UIApplication.shared.open(NSURL(string:"https://vk.com/fukakasiama")! as URL)
+    }
```

Comment:
@Kasiama I know, that you are totally sure, that this is URL, but could you please release it with optional with our force unwrapping?

Reply:
okey, no problem)